lib/galaxy/datatypes/converters/bam_to_bigwig_converter.xml

```diff
@@ -4,15 +4,17 @@
         <requirement type="package">ucsc_tools</requirement>
         <requirement type="package">bedtools</requirement>
     </requirements>
-    <command>
-        bedtools genomecov -bg -split -ibam $input -g $chromInfo 
+    <command><![CDATA[
+        bedtools genomecov -bg -split -ibam '$input' -g '$chromInfo'
+
+        | LC_COLLATE=C sort -k1,1 -k2,2n
 
         ## Streaming the bedgraph file to wigToBigWig is fast but very memory intensive; hence, this
         ## should only be used on systems with large RAM.
-        ## | wigToBigWig stdin $chromInfo $output
+        ## | wigToBigWig stdin '$chromInfo' '$output'
```

Comment:
Actually, the line as is seems to work, so I'm inclined to leave it. `wigToBigWig claims to support bedGraph files (from help):

    wigToBigWig v 4 - Convert ascii format wig file (in fixedStep, variableStep
    or bedGraph format) to binary big wig format.